src/components/AmplitudeAnalytics/constants.ts

```diff
@@ -110,6 +110,7 @@ export enum ModalName {
  * Use to identify low-level components given a TraceContext
```

Comment:
probably remove these changes

Reply:
I was getting build errors without them. Let me try to check why that was happening